gnomad/sample_qc/pipeline.py

```diff
@@ -282,6 +310,9 @@ def annotate_sex(
         logger.info("Filtering to provided sites")
         mt = mt.annotate_rows(**sites_ht[mt.row_key])
         mt = mt.filter_rows(hl.is_defined(mt[aaf_expr]))
+        if is_vds:
```

Comment:
Is this needed? If only keeping `mtds.variant_data` and not using `mtds` after this, I don't think `hl.vds.filter_variants` is needed